core/src/com/po/fuck/model/weapons/AnimatedBullet.java

```diff
@@ -0,0 +1,26 @@
+package com.po.fuck.model.weapons;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.badlogic.gdx.graphics.g2d.Animation;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+
+public class AnimatedBullet extends Bullet {
+    protected static final Map<Class<?>,Animation<TextureRegion>> animationMap = new HashMap<>();
+
+    protected AnimatedBullet(float life_time){
+        super(life_time);
```

Comment:
Could you please state your opinion on initializing everything in initializer blocks? I understand that this could prevent writing repeated code for different constructors. However, initialization of final variables is not allowed in such blocks in Java.